api.md

```diff
@@ -2148,6 +2148,13 @@ WHERE table_schema='public' AND table_name='metrics';
 (1 row)
 ```
 
+If you want to see the current interval length for your hypertables, you can check the `_timescaledb_catalog` as follows. Note that for time-based interval lenghts, these are reported in microseconds.
+
+```Sql
```

Comment:
I _think_ these language designations might need to be all lowercase "sql"

Reply:
You're right - fixing these.